### ISSUE TITLE
Home: Contrast ratio fails on the focus for the Find button

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#11]
URL: https://hcdev.openplus.ca/en
User agent: Firefox
Language: both

**Issue description** The contrast between the focus and the adjacent colours is not enough for the Find button in Firefox.  The focus contrast with the outer blue is 1.16:1.
![image](https://user-images.githubusercontent.com/65728700/126992451-76bb8bc7-72c1-42a8-92d8-11de8e6ce0e0.png)

**steps to reproduce:** Tabbing with Keyboard

**Expected results**: The contrast should be minimum 3:1 with the background so that it's easier to see.

**Comment by ndolter** [comment]
Validated that the contrast ratio in Firefox is now 3:7:1 in both French and English.
